code/datums/elements/turf_transparency.dm

```diff
@@ -0,0 +1,66 @@
+/datum/element/turf_transparency
+	var/show_below = FALSE
+
+///This proc sets up the signals to handle updating viscontents when turfs above/below update. Handle plane and layer here too so that they don't cover other obs/turfs in Dream Maker
+/datum/element/turf_transparency/Attach(datum/target, show_below)
+	. = ..()
+	if(!isturf(target))
+		return ELEMENT_INCOMPATIBLE
+
+	var/turf/our_turf = target
+
+	src.show_below = show_below
+
+	our_turf.plane = OPENSPACE_PLANE
+	our_turf.layer = OPENSPACE_LAYER
```

Comment:
This one not best place. It must be in Initialize()

Don't even, but, it maybe break glass floor.

Reply:
This would run in (Late)Initialize

Reply:
I mean in turf init, not in element.

Reply:
shouldnt matter much: It's better to let the element handle it though because its part of transparency behavior